app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -18,7 +18,7 @@ def self.params_for_create(ems)
           :component  => 'text-field',
           :name       => 'size',
           :id         => 'size',
-          :label      => _('Size (in bytes)'),
+          :label      => _('Size (GB)'),
           :type       => 'number',
           :step       => 1024 * 1024 * 1024,
```

Comment:
@skateman I'm confused as to what `:step` here defines. The resulting options hash passed to `self.raw_create_volume` ends up with the value set by the user (no bytes->GB conversion or anything).

Reply:
Other providers require here a size in bytes, the old form consumed gigabytes and then scaled it up in the JS code. I basically eliminated this by going back to bytes and the `step` param is for the [arrow buttons in the numeric input](https://www.w3schools.com/tags/att_input_step.asp).

Reply:
If your backend goes with gigabytes, it's fine, just please drop the `step` attribute here...

Reply:
That makes sense. Our backend does use GB so I pushed a commit to drop `step`. Thanks @skateman !